tools/kagami/src/main.rs

```diff
@@ -45,7 +46,9 @@ trait RunArgs<T: Write> {
 #[command(name = "kagami", version, author)]
 enum Args {
     /// Generate cryptographic key pairs using the given algorithm and either private key or seed
-    Crypto(Box<crypto::Args>),
+    /// Or sign genesis block offline
+    #[clap(subcommand)]
+    Crypto(CryptoMode),
```

Comment:
```suggestion
    Crypto(crypto::Args),
```

This would align with the surrounding style, imo.